proxywasm/proxytest/http.go

```diff
@@ -27,9 +27,10 @@ type (
 		httpStreams map[uint32]*httpStreamState
 	}
 	httpStreamState struct {
-		requestHeaders, responseHeaders   [][2]string
-		requestTrailers, responseTrailers [][2]string
-		requestBody, responseBody         []byte
+		requestHeaders, responseHeaders       [][2]string
+		requestTrailers, responseTrailers     [][2]string
+		requestBodyBuffer, responseBodyBuffer []byte
+		requestBody, responseBody             []byte
```

Comment:
comment on what are the difference between requestBody vs requestBodyBuffer?

Reply:
Sure, do they make sense?

Reply:
💯